src/components/SelectAvatar/index.js

```diff
@@ -37,14 +39,27 @@ const SelectAvatar = ({ setAvatarData }) => {
 
   return (
     <>
-      <Image source={avatarSource || defaultProfileImage} style={styles.image} />
-      <Button handleOnPress={onSelect} title={selectAvatar} titleColor={BLACK} />
+      <View style={styles.container}>
+        <Image source={profileCircles} />
+        {avatarUrl && !avatarSource ? (
+          <Image source={{ uri: avatarUrl }} style={styles.profileImage} />
+        ) : (
+          <Image source={avatarSource || defaultProfileImage} style={styles.profileImage} />
+        )}
```

Comment:
I would suggest making a separate component for the avatar profile so you can pass as source avatarSource or avatarUrl

```suggestion
    <Avatar source={{ uri: avatarSource || avatarUrl }} >
```
for this to work you need to do `setAvatarSource(uri)` (the value not the object) on `onSelect ` function.

In this way, you send `avtarSource` if the user selects something from the library or `avatarUrl` in the other case.  Then, you can have`defaultProfileImage` as a default prop for the source on Avatar component. 

WDYT? in order to make this cleaner. You can add this on a next PR no need to change this now 😄